.gitignore

```diff
@@ -1,3 +1,14 @@
+# JetBrains idea
+.idea/*
```

Comment:
I don't like having developer-specific entries in `.gitignore`. I think this should be in your global `~/.gitignore`

Reply:
good, point!